lua/Report.lua

```diff
@@ -101,13 +124,23 @@ Report_ = {
 	get = function(self)
 		local template = {}
 		for i = 1, self.nextIdx_ - 1 do
-			table.insert(template, {text = self.text[i], separator = self.separator[i], image = self.image[i], heading = self.heading[i]})
+			table.insert(template, {
+				text = self.text[i],
+				separator = self.separator[i],
+				image = self.image[i],
+				heading = self.heading[i],
+				tabela = self.tabela[i], -- New function teste
```

Comment:
Translate to English

Reply:
in this case I need to use other name, the name (table) conflicts with another variable of the same name already existing in the system.